test/testInteraction.cpp

```diff
@@ -568,12 +568,11 @@ TEST(EMPairProduction, secondaries) {
 	// loop over photon backgrounds
 	for (int f = 0; f < fields.size(); f++) {
 		m.setPhotonField(fields[f]);
-		
-		// loop over energies Ep = (1E10 - 1E23) eV
-		for (int i = 0; i < 130; i++) {
+		for (int i = 0; i < 130; i++) { // loop over energies Ep = (1e10 - 1e23) eV
 			double Ep = pow(10, 10.05 + 0.1 * i) * eV;
 			Candidate c(22, Ep);
 			c.setCurrentStep(std::numeric_limits<double>::max());
```

Comment:
Does it make sense to use here and in the other test the same value for the currentStep? This of course should then be the smallest one: 1e3Mpc, which is probably large enough for all simulation setups anyway, isn't it?

Reply:
For pair production it doesn't really matter which step is used and the tests converge fast.
For inverse Compton scattering and triplet pair production, if the step is very large it takes ages to finilise the test. A suitable value is ~1e4 Mpc (Hubble horizon).

Is your suggestion to change this step to the same value for all EM processes?

Reply:
Yes, that was my suggestion.